client/post-editor/post-editor.jsx

```diff
@@ -82,6 +81,7 @@ import { removep } from 'lib/formatting';
 import QuickSaveButtons from 'post-editor/editor-ground-control/quick-save-buttons';
 import EditorRevisionsDialog from 'post-editor/editor-revisions/dialog';
 import PageViewTracker from 'lib/analytics/page-view-tracker';
+import { pauseGuidedTour } from 'state/ui/guided-tours/selectors';
```

Comment:
I think this should be in the actions folder?

```suggestion
import { pauseGuidedTour } from 'state/ui/guided-tours/actions';
```

Reply:
Why is there no face palm emoji :) 🌴 🤕 ?? :)

Thanks!!